tests/template/srw_shadow_data/to_shadow_grating.out/shadow.json

```diff
@@ -83,7 +83,7 @@
             "offx": 0,
             "offy": 0,
             "offz": 0,
-            "order": -1.0,
+            "order": 1,
             "param": 0.0,
```

Comment:
This test case is a little odd that it changed sign from -1.0 to 1. Could you look at the conversion in srw_shadow.py and make sure it is doing the right thing?

Reply:
In `srw_shadow.py`, if the type for the field is Float in the schema then the value will be multiplied by `scale` which is -1 for diffractionOrder:

```python
if schema[from_f][1] == "Float":
        v = float(v) * scale
```

But the type was changed to Integer. Do we still want to scale by -1 now that it is integer?

Reply:
Yes, I think that should be changed to check if it is Float or Integer, apply the scale. Shadow needs the diffraction order to be negative to agree with SRW.